Library/PhantomLib/CustomControls/FloatingLabel.xaml.cs

```diff
@@ -46,6 +49,14 @@ static void UltimateEntry_Changed(BindableObject bindable, object oldValue, obje
             }
         }
 
+        private void _ultimateEntry_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            if(!string.IsNullOrEmpty(e.NewTextValue))
```

Comment:
Seems like this would animate every time someone types in text. Is that not the case?

Reply:
yeah, it does call the code every time which is undesirable
i added a check to see if the label is already floating